docs/src/resources/two-side-column.md

```diff
@@ -0,0 +1,923 @@
+    val in  = Input(UInt(4.W))
+    val out = Output(UInt(4.W))
+  })
+
+val two  = 1 + 1
+val utwo = 1.U + 1.U
+
+io.out := io.in
+}
+```
+```scala mdoc:invisible
+ChiselStage.emitVerilog(new MyModule)
+```
+</td>
+         </tr>
+    </table>
+<html>
+<body>
+
+# Wire assignment
+
+<html>
+<body>
+    <table border ="0">
+          <tr>
+            <td><b style="font-size:30px">Verilog</b></td>
+            <td><b style="font-size:30px">Chisel</b></td>
+         </tr>
+         <tr>
+<td>
+
+```
+wire [31:0] 
+a = 32'd42; 
+wire [31:0] 
+b = 32'hbabecafe; 
+wire [15:0] c; 
+assign c = 16'b1;
+```
+
+</td>
+<td>
+
+```scala mdoc:silent
+
+
+class MyWireAssignmentModule extends Module {
+ val a = WireDefault(42.U(32.W))
+ val b = WireDefault("hbabecafe".U(32.W))  
+ val c = Wire(UInt(16.W)) 
+  c := "b1".U;
+}
+```
+```scala mdoc:invisible
+ChiselStage.emitVerilog(new MyWireAssignmentModule)
+```
+</td>
+         </tr>
+<tr>
+<td>
+
+```
+module MyWireAssignmentModule(
+  input   clock,
+  input   reset
+);
+endmodule
+```
+
+
+</td>
+<td>
+
+```scala mdoc:silent
+class MyWireAssignmentModule2 extends Module {
+ val a = WireDefault(42.U(32.W))
+ val aa = 42.U(32.W)
+ val b = WireDefault("hbabecafe".U(32.W))  
+ val c = Wire(UInt(16.W)) 
+ val d = Wire(Bool())
+ val e = Wire(UInt())
+ val f = WireDefault("hdead".U)
+ val g = Wire(SInt(64.W))
+ val h = WireDefault(5.asSInt(32.W))
+ val i = WireDefault((3.S(16.W)))
+
+c := "b1".U;
+d := true.B
+e := "b1".U(3.W);
+g := -5.S
+}
+```
+```scala mdoc:invisible
+ChiselStage.emitVerilog(new MyWireAssignmentModule2)
+```
+
+</td>
+</tr>
+    </table>
+<html>
+<body>
+
+# Register assignment
+
+<html>
+<body>
+    <table border ="0">
+          <tr>
+            <td><b style="font-size:30px">Verilog</b></td>
+            <td><b style="font-size:30px">Chisel</b></td>
+         </tr>
+         <tr>
+<td>
+
+```
+module RegisterModule(
+  input         clock,
+  input         reset,
+  input  [11:0] io_in,
+  output [11:0] io_out
+);
+  assign io_out = io_in; // @[main.scala 18:10]
+endmodule
+```
+</td>
+<td>
+
+```scala mdoc:silent
+class RegisterModule extends Module {
+  val io = IO(new Bundle {
+    val in  = Input(UInt(12.W))
+    val out = Output(UInt(12.W))
+  })
+
+  val registerWithInit = RegInit(42.U(12.W))
+  registerWithInit := registerWithInit - 1.U
+  io.out := io.in
+}
+```
+```scala mdoc:invisible
+ChiselStage.emitVerilog(new RegisterModule)
+```
+</td>
+         </tr>
+    </table>
+<html>
+<body>
+
+# Case statement
+
+<html>
+<body>
+    <table border ="0">
+          <tr>
+            <td><b style="font-size:30px">Verilog</b></td>
+            <td><b style="font-size:30px">Chisel</b></td>
+         </tr>
+         <tr>
+<td>
+
+```
+module CaseStatementModule(
+  input        clock,
+  input        reset,
+  input  [2:0] a,
+  input  [2:0] b,
+  input  [2:0] c,
+  input  [1:0] sel,
+  output [2:0] out
+);
+  wire [2:0] _GEN_0 = 2'h2 == sel ? c : 3'h0; // @[main.scala 16:16 28:8 14:9]
```

Comment:
This needs to be rewritten to be more "how a verilog person would write it"... @mwachs5 to find the link from the original souce we were using